engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetBlend.java

```diff
@@ -15,48 +15,55 @@
  */
 package org.terasology.rendering.dag.stateChanges;
 
+import static org.lwjgl.opengl.GL11.GL_BLEND;
 import org.terasology.rendering.dag.RenderPipelineTask;
 import org.terasology.rendering.dag.StateChange;
+import org.terasology.rendering.dag.tasks.DisableCapabilityTask;
 import org.terasology.rendering.dag.tasks.EnableCapabilityTask;
 
 /**
  * TODO: Add javadocs
  */
-public class EnableCapability implements StateChange {
-    private int capability;
-    private EnableCapabilityTask task;
+public final class SetBlend extends SetCapability {
```

Comment:
Hmmm, I really don't see the conflict. Can you explain it more, please? My rationale was, from OpenGL Documentation for `glEnable`:

![screen shot 2016-08-09 at 22 30 56](https://cloud.githubusercontent.com/assets/136392/17530621/0244fbfa-5e81-11e6-802c-b876135e4013.png)

For your second comment, I really don't see any good reason why we need someting like that with boolean parameter. But I am willing to create a subclasses for SetBlend:

* EnableBlend extends SetBlend(true)
* DisableBlend extends SetBlend(false)

which would be more better, I think.